assets/scss/modules/_card.scss

```diff
@@ -54,6 +59,24 @@
       margin: 10px 0 0;
       font-size: 14px;
     }
+
+    .dp-btn-free {
+      justify-content: center;
+      display: flex;
+      align-items: flex-end;
+      height: 170px;
```

Comment:
Shouldn't all buttons have the same height? or standarized height?

Reply:
This is not a button it only seems like it, but you're right maybe we can push it with another element 👍